metatron-app/metatron-discovery
# fix metadata error

### Description

1. The code table popup does not move to the detail screen
2. Loading does not disappear after entering more than 150 characters in the metadata detail screen.
3. Metadata not deleted
4. Explore data bookmark display problem
5. Catalog Tab Menu Appears
6. Name Display Location Issues in the Term Dictionary
7. Metadata list design change issue
8. Problem that input description more than 50 characters in term dictionary detail screen
9. Problem with validation not being made after entering input when creating term dictionary
10. Explore data search list date display error

**Related Issue** : 
METATRON-2976
METATRON-2969
METATRON-2958
METATRON-2952
METATRON-2950
METATRON-2978
METATRON-2955
METATRON-2982
METATRON-2989
METATRON-2949

### How Has This Been Tested?
1. Make sure you go to the Details screen in the Code Table pop-up.
2. On the metadata detail screen, enter a description of 150 characters or more and check that the loading is normally lost.
3. Confirm that the metadata is deleted.
4. Confirm that the Explore data favorites area is gone.
5. Make sure the Catalog tab menu is hidden.
6. In the glossary, edit the name in the top bar and verify that it is missing from the content.
7. Verify that the metadata list is the same as the Explore data search list.
8. On the Term Dictionary Details screen, verify that your description is longer than 50 characters.
9. When you create a term dictionary, verify that the validation check runs after you finish typing.
10. In the Explore data search list, verify that the date is displayed as the modified date.

#### Need additional checks?

### Types of changes


### Checklist:


### Additional Context
